zebrad/tests/common/lightwalletd/send_transaction_test.rs

```diff
@@ -202,15 +198,14 @@ pub async fn run() -> Result<()> {
              in isolated mempool: {tx:?}",
         );
 
-        _counter += 1;
+        counter += 1;
     }
 
-    // TODO: This check sometimes works and sometimes it doesn't so we can't just enable it.
-    // https://github.com/ZcashFoundation/zebra/issues/7529
-    //assert!(
-    //    counter >= 1,
-    //    "all transactions from future blocks failed to send to an isolated mempool"
-    //);
+    // This check will fail if there are no non-coinbase transactions in the first 50 non-finalized blocks
+    assert!(
+        counter >= 1,
+        "all transactions from future blocks failed to send to an isolated mempool"
+    );
 
     // GetMempoolTx: make sure at least one of the transactions were inserted into the mempool.
     tracing::info!("calling GetMempoolStream gRPC to fetch transactions...");
```

Comment:
Optional: Does updating the service file also allow the `GetMempoolStream` check to be re-enabled?

Reply:
No, but the other assertion failed and had the message, "developers: should fail if `get_mempool_stream` start working": 

https://github.com/ZcashFoundation/zebra/pull/7600/files#diff-91452b1b52c32dc72de4dd903dd355f10ffc99e30d7217df595bb0177f88bd27L234-L237